Dashboard/src/app/services/dashboard-data.service.ts

```diff
@@ -1,94 +1,14 @@
-import {Injectable, Injector} from '@angular/core';
-import {async, fakeAsync, tick} from '@angular/core/testing';
-import {BaseRequestOptions, ConnectionBackend, Http, RequestOptions} from '@angular/http';
-import {Response, ResponseOptions} from '@angular/http';
-import {MockBackend, MockConnection} from '@angular/http/testing';
-
-//move this part into separate file later
-const data = {
-  nowPlaying: {
-    artist: 'Eagles',
-    song: 'Witchy Woman',
-    album: 'Eagles (Remastered)',
-    albumUri: 'http://cont-2.p-cdn.com/images/public/rovi/albumart/9/2/3/2/075596062329_500W_500H.jpg',
-    genre: 'Santana Radio',
-    loved: true,
-    playedSeconds: 0,
-    durationSeconds: 252
-  },
-  currentInput: 'Pandora',
-  telemetryData: {
-    temperature: 22.8,
-    humidity: 68.8,
-    heatIndex: 22.93,
-    gasValue: 308,
-    measured: '2017-11-06T21:44:22.001387Z'
-  },
-  isTurnOn: true,
-  volume: -245
-}
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import {Observable} from 'rxjs/Rx';
 
 @Injectable()
 export class DashboardDataService {
 
-  constructor( private http: Http ) { }
-
-  public getData() {
-    //change api later
-    return this.http.get('myservices.de/api/heroes')
-    .toPromise()
-    .then(response => response.json().data)
-    .catch(e => this.handleError(e));
-  }
+  constructor(private http: Http) { }
 
-  private handleError(error: any): Promise<any> {
-    console.error('An error occurred', error);
-    return Promise.reject(error.message || error);
+  public getData(): Observable<object> {
+    return this.http.get('/fake-backend/data')
```

Comment:
base address from url should be in environment file... For example
`this.httpClient.get(`${environment.apiUrl}$/api/SmartHouse/Dashboard`)`